indigo_app/views/works.py

```diff
@@ -534,6 +540,8 @@ def get_works(self, table):
                     work.publication_number = row.get('publication_number')
                     work.created_by_user = self.request.user
                     work.updated_by_user = self.request.user
+                    if not row.get('primary'):
```

Comment:
Prefer `work.stub = not row.get('primary')`

Reply:
that is much better thank you!